app/models/label/LabelTable.scala

```diff
@@ -785,6 +802,20 @@ object LabelTable {
     )
   }
 
+  /**
+    * Returns a LabelMetadataWithValidation object that has the label properties, tags, and includes validation results.
+    *
+    * @param label label from query
+    * @param tags list of tags as strings
+    * @return LabelMetadata object
+    */
+  def labelAndTagsToLabelMetadataWithValidation(label: (Int, String, Boolean, String, Float, Float, Int, Int, Int, Int, Int, Int, String, String, Option[java.sql.Timestamp], String, String, Option[Int], Boolean, Option[String],Map[String,Int]), tags: List[String]):
+  LabelMetadataWithValidation = {
+      LabelMetadataWithValidation(label._1, label._2, label._3, label._4, label._5, label._6, label._7, label._8,
```

Comment:
I think the fix you made for the other function (accidentally indenting by 4), you accidentally did incorrectly here! Probably a copy/paste error